packages/ses/src/ses.d.ts

```diff
@@ -0,0 +1,38 @@
+type safetyPreference = 'safe' | 'unsafe';
+
+type LockdownOpts = {
+  fringeSet?: {
+    add?: Function,
+    has?: Function,
+  },
+  dateTaming?: safetyPreference,
+  errorTaming?: safetyPreference,
+  mathTaming?: safetyPreference,
+  regExpTaming?: safetyPreference,
+
+  [extraOption:string]?: any,
+};
+
+declare var lockdown = (LockdownOpts) => void;
+
+declare var harden = (any) => any;
+
+declare class Compartment {
+  constructor(endowments: Object);
+  evaluate(command: string): any;
+  globalThis: any;
+  import: () => Promise<any>;
+  importNow: () => any;
+  module: Function;
+}
+
+declare var StaticModuleRecord = Object;
```

Comment:
This is a class constructor that accepts `(moduleSource: string, moduleLocation:string)`.

The class is an opaque object. The strongest way to depict the type is that the StaticModuleRecord can only be created through that constructor function, and the `importHook` `Compartment` constructor function must return a `Promise<StaticModuleRecord>` so-created.

Reply:
`StaticModuleRecord` was also in the tc-39 document, but I've added a `constructor` definition for the string parameters you described.